Controllers/UserController.cs

```diff
@@ -109,6 +108,67 @@ public IHttpActionResult GetAll()
             return Ok(users);
         }
         //------------------------------------------------------------------------------
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("auth")]
```

Comment:
Methods attributes can be declared on this way which looks cleaner

```suggestion
 [HttpPost, AllowAnonymous, Route("auth")]
```